ext/neural_snow/DataTools.jl

```diff
@@ -542,6 +633,1171 @@ function snotel_metadata(;
+        h2 = data[!, :date] .< Date("2014-07-01")
+        w1 = data[!, :date] .>= Date("2022-06-01")
+        sflags[s1 .| s2] .= true
+        hflags[h1 .| h2] .= true
+        wflags[w1] .= true
+
+    elseif data[1, :id] .== 969
+        f1 = Date("2008-06-01") .<= data[!, :date] .< Date("2009-06-01")
+        s1 = Date("2016-05-01") .<= data[!, :date] .< Date("2016-09-01")
+        s2 = data[!, :date] .< Date("2006-10-01")
+        s3 = data[!, :date] .>= Date("2022-06-01")
+        s4 = Date("2016-04-01") .<= data[!, :date] .< Date("2016-08-01")
+        w1 = Date("2016-05-01") .<= data[!, :date] .< Date("2017-10-01")
+        w2 = Date("2020-08-01") .<= data[!, :date] .< Date("2021-02-01")
+        sflags[s1 .| s2 .| s3 .| s4 .| f1] .= true
+        hflags[f1] .= true
+        wflags[w1 .| w2] .= true
+
+    elseif data[1, :id] .== 974
+        w1 = data[!, :date] .< Date("2010-09-01")
+        wflags[w1] .= true
+
+    elseif data[1, :id] .== 978
+        s1 = data[!, :date] .>= Date("2018-08-01")
+        sflags[s1] .= true
+
+    elseif data[1, :id] .== 1030
+        h1 = Date("2016-09-01") .<= data[!, :date] .< Date("2017-09-01")
+        w1 = Date("2012-01-01") .<= data[!, :date] .< Date("2013-06-01")
+        w2 = Date("2021-06-01") .<= data[!, :date] .< Date("2022-10-01")
+        s1 = data[!, :date] .< Date("2005-01-01")
+        s2 = data[!, :date] .>= Date("2022-06-01")
+        w3 = data[!, :date] .< Date("2006-06-01")
+        sflags[s1 .| s2] .= true
+        hflags[h1] .= true
+        wflags[w1 .| w2 .| w3] .= true
+
+    elseif data[1, :id] .== 1053
+        s1 = data[!, :date] .>= Date("2020-06-01")
+        s2 = Date("2006-06-01") .<= data[!, :date] .< Date("2007-09-01")
+        sflags[s1 .| s2] .= true
+
+    elseif data[1, :id] .== 1083
+        s1 = data[!, :date] .>= Date("2023-06-01")
+        f1 = Date("2012-10-25") .<= data[!, :date] .< Date("2012-11-24")
+        sflags[s1 .| f1] .= true
+        hflags[f1] .= true
+
+    elseif data[1, :id] .== 1105
+        s1 = data[!, :date] .>= Date("2012-06-01")
+        sflags[s1] .= true
+
+    elseif data[1, :id] .== 1170
+        f1 = data[!, :date] .>= Date("2023-05-01")
+        sflags[f1] .= true
+        wflags[f1] .= true
+
+    elseif data[1, :id] .== 1254
+        f1 = data[!, :date] .>= Date("2023-05-01")
+        sflags[f1] .= true
+        hflags[f1] .= true
+
+    elseif data[1, :id] .== 1286
+        s1 = Date("2021-06-01") .<= data[!, :date] .< Date("2021-10-01")
+        sflags[s1] .= true
+
+    elseif data[1, :id] .== 963
+        s1 = data[!, :date] .< Date("2007-06-01")
+        sflags[s1] .= true
+
+    elseif data[1, :id] .== 1035
+        s1 = Date("2009-06-01") .<= data[!, :date] .< Date("2013-06-01")
+        sflags[(!).(s1)] .= true
+
+    elseif data[1, :id] .== 1092
+        s1 = data[!, :date] .> Date("2018-06-01")
+        sflags[s1] .= true
+    end
+    return sflags, hflags, wflags
+end
```

Comment:
Which of these filters did you end up using in the paper?

Reply:
If you are referring to "which of the three" between sflags, hflags, and wflags, I use all three (these are filters for the respective data columns, not the entire dataframe at once), see their application in bcqc_hourly(). If you are asking which of the site-level filters are utilized, all listed site filters are applied to the paper data.